app/src/main/java/com/google/android/samples/socialite/ui/home/timeline/TimelineViewModel.kt

```diff
@@ -63,6 +65,15 @@ class TimelineViewModel @Inject constructor(
         }
     }
 
+    // Used to track performance of preload manager
+    private val firstFrameRenderedListener = object : Player.Listener {
+        override fun onRenderedFirstFrame() {
+            super.onRenderedFirstFrame()
+            val timeToFirstRenderFrame = System.currentTimeMillis() - timeAtPlayerPrepare
+            // Use this value in future to compare performance with and without preload manager
```

Comment:
would it be useful to add a log statement here to print out this value?

Reply:
Ack. Added a log statement